src/api/local-storage.js

```diff
@@ -0,0 +1,64 @@
+class localStorageApi {
+
+  static getUsers() {
+    var existingUsers = JSON.parse(localStorage.getItem("allStoredUsers")) || [];
+    console.log("getUsers() current existing stored users = ", existingUsers);
+    return existingUsers;
+ }
+ 
+  static createUser(newUser) {
+    var existingUsers = localStorageApi.getUsers();
+    existingUsers.push(newUser);
+    localStorage.setItem("allStoredUsers", JSON.stringify(existingUsers));
+    console.log("createUser() new current existing users = ", existingUsers);
+    return existingUsers;
+  }
+
+  static getUser(userId) {
+    var existingUsers = localStorageApi.getUsers();
+    var found = existingUsers.find(allStoredUsers => {
```

Comment:
Unless this is just semi-crappy naming, I'm going to guess you're missing something here. `allStoredUsers` makes it seem like the entire array of existing users is being passed in each loop, when (afaict) it's not. It'd be better named something like

```suggestion
    var found = existingUsers.find(storedUser => {
```

or even 

```suggestion
    var found = existingUsers.find(user => {
```

Reply:
furthermore, you could just return `existingUsers.find(...)` since you're not manipulating it any further after setting it. You also don't need to include the curly brackets (`{` and `}`) or the enclosed `return` if the function is a one line return statement:
```suggestion
    return existingUsers.find(user => user.id === id);
```